src/Step.js

```diff
@@ -22,14 +27,25 @@ const Step = ({ children, name, validator, autoSkip }: Props) => {
     if (!initialized) {
       registerStep(stepInfo);
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Isn't it sufficient to pass `registerStep` as a dependency to re-enable the rule?

Reply:
The linter would still complain about lacking the "stepInfo" prop. When I tried adding the register and stepInfo prop, the Step ended up executing in an infinite loop, so I chose to not change the implementation of the existing useEffect(s), because they currently work and we can fix in the future if need be.

Reply:
Fixed in b0ed945

Reply:
> The linter would still complain about lacking the "stepInfo" prop. 

The infinite loop is because you create `stepInfo` as a new object all times. You can fix that by passing all three params as dependencies to `useEffect`.

```
useEffect(() => {
  // ...
}, [name, validator, autoSkip]);
```

Reply:
I was able to fix one of the `react-hooks/exhaustive-deps` issues with this!

Unfortunately the second useEffect is still suffering from this issue. 

```
  useEffect(() => {
    if (initialized) {
      updateStep({
        name,
        validator,
        autoSkip,
      });
    }
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [autoSkip, validator, updateStep, initialized]);
```

updateStep() is causing a re-render of Wizard which is triggering the useEffect infinitely.

Some of the ways we can fix this is:
- converting Wizard to a class based component
- continue to disable eslint for this line
- @kmkr Was there another option? I can't remember anymore...

@rix1 Do you have any tips and tricks here?

Reply:
@rix1 suggests that a solution might be found using this pattern: https://github.com/otovo/react-losen/pull/24#discussion_r367483069

This needs further investigation